tests/integration/test_vulnerability_detector/test_scan_results/test_scan_nvd_feeds.py

```diff
@@ -24,46 +24,55 @@
 current_test_path = os.path.dirname(os.path.realpath(__file__))
 test_data_path = os.path.join(current_test_path, 'data')
 configurations_path = os.path.join(test_data_path, 'wazuh_nvd_configuration.yaml')
-vulnerabilities_data_path = os.path.join(test_data_path, CUSTOM_NVD_VULNERABILITIES_1)
+vulnerabilities_data_path = os.path.join(test_data_path, vd.CUSTOM_NVD_VULNERABILITIES_1)
+custom_cpe_helper_data_path = os.path.join(test_data_path, vd.CUSTOM_CPE_HELPER)
+custom_msu_data_path = os.path.join(test_data_path, vd.CUSTOM_MSU)
 
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
 SCAN_TIMEOUT = 40
 
 
+compress_gzip_file(custom_msu_data_path, vd.MSU_PATH)
+
+copy(custom_cpe_helper_data_path, vd.CPE_HELPER_PATH)
+
 # Set configuration
-parameters = [{'NVD_JSON_PATH': os.path.join(test_data_path, CUSTOM_NVD_FEED)}]
-metadata = [{'nvd_json_path': os.path.join(test_data_path, CUSTOM_NVD_FEED)}]
+parameters = [{'NVD_JSON_PATH': os.path.join(test_data_path, vd.CUSTOM_NVD_FEED)}]
```

Comment:
Change NVD_JSON_PATH to lower case

Reply:
This is the convention we are using for naming the keys of these maps, parameters in uppercase and metadata in lowercase, here you have another example:
https://github.com/wazuh/wazuh-qa/blob/77223ed24d8fff611947e817d180bf744720ed4c/tests/integration/test_vulnerability_detector/test_providers/test_providers_os.py#L22-L45